packages/website/src/pages/docs/Ecosystem/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
   "Tooling": "Tooling",
   "Gallery": "Gallery",
-  "LanguageModels": "Language Models"
+  "LanguageModels": "Language Models",
+  "LLMPrompt": "LLM Prompt"
```

Comment:
This page is not working, even after a full Vercel rebuild with no cache.

My guess is because of the order in `package.json`, you do `next build` first and generate the mdx only after that.